javacutil/src/main/java/org/checkerframework/javacutil/AnnotationUtils.java

```diff
@@ -553,10 +553,8 @@ public static boolean hasInheritedMeta(AnnotationMirror anno) {
             case TYPE_USE:
                 return EnumSet.noneOf(ElementKind.class);
             default:
-                // TODO: Add actual case to check for the enum constant and return Set containing
```

Comment:
I think the original `TODO` was to actually use a `case MODULE:`. We are using the Java 9 compiler, so the constant should be available. When we tried that, some configuration didn't work. Maybe that configuration is fixed by now and we can actually use the proper `case`?

Reply:
No, we are using the Java 9 compiler, but still using the Java 8 version of `ElementType`.

Reply:
The `TODO` comment should remain, to mark this as a place to change when we require a Java 9+ JVM. Isn't that what the original TODO was for? Or should we introduce a different TODO comment?

Reply:
@wmdietl I added this `TODO`.  Will this suffice?

```java
// TODO: Use MODULE enum constants directly instead of looking them up by name.
// (Java 11)
```

Looking back, I think the original TODO had two components:
* Add case for `ElementType.MODULE` instead of looking it up by name in the default case.
* Return a set containing `ElementKind.MODULE` instead of returning an empty set.

This PR addresses 2 (which I think is the more important one) but not 1 (which I missed in my original reading).

Reply:
Yes, that sounds right.